include/constants/battle_string_ids.h

```diff
@@ -382,5 +382,7 @@
 #define STRINGID_PKMNBOXLANETTESPCFULL      378
 #define STRINGID_TRAINER1WINTEXT            379
 #define STRINGID_TRAINER2WINTEXT            380
+#define STRINGID_PKMNPICKEDUPITEM           381
+#define STRINGID_PKMNPICKEDUPITEMS          382
```

Comment:
Unnecessary

Reply:
this was an accident i am trying to undo. I meant for this to be in a new branch